packages/cli/src/prebuild.ts

```diff
@@ -489,15 +489,30 @@ export const prebuild = async (options: {
     atomic: siteData.build.pages.compiler?.atomicStyles ?? true,
   });
 
-  await ensureFileInPath(join(generatedDir, "index.css"), cssText);
+  await createFileIfNotExists(join(generatedDir, "index.css"), cssText);
 
   spinner.text = "Generating routes and pages";
 
-  const routeTemplatePath = normalize(join(cwd(), "app/routes/template.tsx"));
+  // MARK: - Route templates read
+  const routeTemplatesDir = join(cwd(), "app/route-templates");
 
-  const routeFileTemplate = await readFile(routeTemplatePath, "utf8");
+  const routeTemplatePath = normalize(
+    join(routeTemplatesDir, "html-doc-type.tsx")
```

Comment:
html-template?

Reply:
or just html.tsx

Reply:
Moved to `route-template` dir. So removed `template` keyword.
Renamed to just html, xml